frame/wormhole/backing/s2s/src/lib.rs

```diff
@@ -137,6 +149,20 @@ pub mod pallet {
 	#[pallet::pallet]
 	pub struct Pallet<T>(_);
 
+	#[pallet::hooks]
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		fn on_initialize(now: BlockNumberFor<T>) -> Weight {
+			let latest_daily_record = DailyUnlocked::<T>::get().0;
+			if latest_daily_record > Zero::zero()
+				&& latest_daily_record < now - now % T::BlocksPerDay::get()
```

Comment:
latest_daily_record is the time of latest record? Need more documents here.

Reply:
Yes, this is the time(it's indicated by blocknumber)